thunder/tests/opinfos.py

```diff
@@ -2531,6 +2531,82 @@ def where_error_generator(op, device, dtype=torch.float32, **kwargs):
 conditional_and_mask_ops.append(where_opinfo)
 
 
+def nan_to_num_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    a = make((4, 4), dtype=dtype, requires_grad=requires_grad)
+    double_max = torch.finfo(torch.float64).max
+    double_min = torch.finfo(torch.float64).max
```

Comment:
This line incorrectly sets `double_min` to the maximum representable double value

Reply:
Thanks for catching that. Fixed!